Resources/Prototypes/_DV/Entities/Objects/Weapons/Guns/Rifles/rifles.yml

```diff
@@ -23,14 +24,14 @@
       - state: mag-0
         map: ["enum.GunVisualLayers.Mag"]
   - type: GunWieldBonus
-    minAngle: -19
-    maxAngle: -16
+    minAngle: -11
+    maxAngle: -19
```

Comment:
max 20 - 19 = 1 min 13 - 11 = 2

max angle is 1
min angle is 2

so uhhh even though the max arc is 1 the minimal angle is 2 this shit dont make sense